HowTos/github_gui.md

```diff
@@ -0,0 +1,59 @@
+# How To add attendee name using GitHub UI
+
+To add our name to the list of attendees we need to create a new `.yml` file in
+the folder `_data/attendees/`. In this tutorial we will see how we can do this
+on GitHub.
+
+First we need to go to the main page of the repository:
+https://github.com/Nikoleta-v3/getting-started-with-open-source.
+
+On the right side we need to click on the button `Add file` and then `Create new file`.
+
+![Add file](../static/create_new_file.png)
+
+When we do this we are forward to a new page which looks like:
+
+![Data](../static/new_file_and_data.png)
+
+Here we need to give the name and location of your new file. Remember we want the file to be in the folder `_data/attendees/`. Then we just need a name for our `.yml` name. How about using your last name(s)?
+
+In the file we will to include our name and we will also include our GitHub handler in case people want to collaborate with us in the future! Under `Edit new file` add the following lines:
+
+```shell
+- name: Name Surname
+    github: handler
+```
+
+where you change `Name`, `Surname` and `handler`.
```

Comment:
```suggestion
In the file we will to include our name and we will also include our GitHub handle (user name) in case people want to collaborate with us in the future! Under `Edit new file` add the following lines:

```shell
- name: Name Surname
    github: handle
```

where you change `Name`, `Surname` and `handle`.
```

Not strongly attached to this suggestion, but I think "handle" is more commonly used than handler

Reply:
😅 hahaha good catch!